server/utils/nunjucksSetup.ts

```diff
@@ -183,5 +144,30 @@ export function registerNunjucks(app?: express.Express): Environment {
     return address ? address.split(', ').filter(line => line.trim().length > 0) : undefined
   })
 
+  njkEnv.addFilter('separatedDataByFieldName', (data: AdditionalConditionData[]) => {
+    const map = new Map()
+    data.forEach(item => {
+      const collection = map.get(item.field)
+      if (!collection) {
+        map.set(item.field, [item])
+      } else {
+        collection.push(item)
+      }
+    })
+    return Array.from(map, ([fieldName, items]) => ({ fieldName, items })).map(groupedDataItems => {
+      return groupedDataItems.items.map((dataItem: AdditionalConditionData) => dataItem.value).join(', ')
+    })
+  })
+
+  njkEnv.addGlobal('getFormResponses', (formResponses: unknown, inputName: string) => {
+    return formResponses ? [formResponses[inputName]].flat() : undefined
+  })
+
+  njkEnv.addGlobal('getValueFromAdditionalCondition', (additionalCondition: AdditionalCondition, inputName: string) => {
+    return additionalCondition
+      ? additionalCondition.data.filter(item => item.field === inputName).map(item => item.value)
+      : undefined
+  })
+
```

Comment:
Are there tests for the newly added filters?

Reply:
OK as a follow-on piece if you want to get this in.

Reply:
In my next PR I want to separate these functions away from nunjucks, and have them as reusable utils. This means they can be used elsewhere in our code (not just nunjucks) and also makes them more easily unit tested. When I do that I will increase coverage on them